independent-publisher-connectors/apiDefinition.swagger.json

```diff
@@ -0,0 +1,873 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "CraftMyPDF",
+    "description": "CraftMyPDF's drag & drop editor lets you design PDF templates in any browser and generate pixel-perfect PDF documents from reusable templates and JSON data.",
+    "contact": {
+      "name": "Troy Taylor",
+      "url": "https://www.hitachisolutions.com",
+      "email": "ttaylor@hitachisolutions.com"
+    }
+  },
+  "host": "api.craftmypdf.com",
+  "basePath": "/v1",
+  "securityDefinitions": {
+    "API Key": {
+      "type": "apiKey",
+      "in": "header",
+      "name": "X-API-KEY"
+    }
+  },
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/create": {
+      "post": {
+        "description": "This action creates a PDF file with JSON data and your template.",
+        "summary": "Create a PDF",
+        "tags": [
+          "PDF Generation API"
+        ],
+        "operationId": "Create",
+        "deprecated": false,
+        "produces": [
+          "application/json"
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "description": "The body.",
+            "schema": {
+              "$ref": "#/definitions/CreateRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns status and output file.",
+            "schema": {
+              "$ref": "#/definitions/ResponseSuccessSingleFile"
+            },
+            "headers": {}
+          }
+        }
+      }
+    },
+    "/create-async": {},

```

Comment:
@troystaylor Please remove empty operation definitions, they would not add any value to your connector.